introduction/modules/modules.md

```diff
@@ -15,41 +18,51 @@ Connectors are Valtimo configurable components that can be used to connect to ex
 framework to create, configure, and use connectors. Certain Valtimo modules include their own connectors that become
 available automatically for configuration when starting a Valtimo instance.
 
+> **_NOTE:_** Connectors will be replaced by Plugins in the future
+
 ### Contract
 
-Contract contains certain interfaces and events that Valtimo uses in several other modules. Apart from this, contract
-sanitizes endpoint output when exceptions occur. A liquibase runner is available that allows other modules to define
-liquibase configuration that is run on startup. Finally, Contract includes several helper classes, like the
-`QueryDialectHelper` bean that can be used in criteria queries to add JSON column clauses to the query.
+Contract contains certain interfaces and events that Valtimo uses in several other modules. Consider it the 'glue'
+between a module and Core.
+
+Apart from this, contract sanitizes endpoint output when exceptions occur. A liquibase runner is available that allows
+other modules to define liquibase configuration that is run on startup. Finally, Contract includes several helper
+classes, like the `QueryDialectHelper` bean that can be used in criteria queries to add JSON column clauses to the
```

Comment:
```suggestion
that can be used in criteria queries to add JSON column clauses to the
query.
```

that can be used in criteria queries to call DBMS specific functions in a generic way (e.g. finding JSON data).